src/modules/robot/arm_solutions/LinearDeltaSolution.cpp

```diff
@@ -133,8 +133,9 @@ bool LinearDeltaSolution::get_optional(arm_options_t& options) {
     options['R']= this->arm_radius;
 
     // don't report these if none of them are set
-    if(this->tower1_offset != 0.0F || this->tower2_offset != 0.0F || this->tower3_offset != 0.0F ||
-       this->tower1_angle != 0.0F  || this->tower2_angle != 0.0F  || this->tower3_angle != 0.0F) {
+//    if(this->tower1_offset != 0.0F || this->tower2_offset != 0.0F || this->tower3_offset != 0.0F ||
```

Comment:
Yeah, I left it as such since this is a point of discussion. I needed a way to query an arm solution for all its available options. It definitely needs another way.